common/src/main/scala/explore/common/TargetObsQueries.scala

```diff
@@ -163,14 +165,13 @@ object TargetObsQueries {
           observations(programId: "p-2", first: 2147483647) {
             nodes {
               id
-              name
-              observationTarget {
+              pointing: observationTarget {
                 type: __typename
                 ... on Target {
-                  target_id: id
+                  targetId: id
```

Comment:
Are you using `targetId` rather than `target_id` to get nicer field names?

Reply:
Exactly. The aliased names will be used to generate the case classes to store results, so we might as well go with Scala style.